src/Resources/assets/styles/_tailwind.scss

```diff
@@ -58,6 +58,31 @@
         @apply text-neutral-500;
     }
 
+    // Badges
+    .whatwedo-utility-badge {
+        @apply inline-flex items-center rounded-full px-2 py-1 text-xs font-medium bg-primary-500 text-white;
+
+        &--success {
+            @apply bg-success-500;
+        }
+
+        &--warning {
+            @apply bg-warning-500;
+        }
+
+        &--error {
+            @apply bg-error-500;
+        }
+
+        &--neutral {
+            @apply bg-neutral-500;
+        }
+
+        &--primary {
+            @apply bg-primary-500;
+        }
+    }
```

Comment:
Honestly, I really think this should be either one of both based, on the Tailwind Documentation:

- A Twig macro with all the Tailwind classes defined in HTML. Because Tailwind says, using `@apply` is bad practice. And writing CSS like that is almost the opposite of what Tailwind wants to solve.
- Classes defined in the Tailwind Config as JSON.

Additionally, if we start to add more of those “utilities” that aren't actually utilities but “components”, one big `_tailwind.scss` is not gonna serve as a maintainable base layer, but we will need individual files per component.

At this point, we really should question the architecture we are building here. This will not scale and Tailwind will not serve us the way it wants.

Reply:
Completely agree. This is how it has been done in the last years and will be done in the next couple of weeks until we have decided for a proper direction. Building parts in one way and other in a different way isn't helpful. We'll do a refactor and fix it all together once things are clear.